index.js

```diff
@@ -438,17 +438,20 @@ class Body extends events.EventEmitter {
     }
     this.ct = ct;
 
+    const buildRegex = (key) => new RegExp(`${key}\\s*=\\s*"([^"]+)"|${key}\\s*=\\s*"?([^";]+)"?`, 'i');
+    const matchKey = (test, key) => test.match(buildRegex(key))?.filter(item => item);
```

Comment:
The `.filter` is there to remove undefined groups when the right possibility matches, leaving the left one undefined.